.travis.yml

```diff
@@ -156,8 +172,8 @@ script:
   - ./ci/suppress_output bash src/ray/test/run_core_worker_tests.sh
   - ./ci/suppress_output bash src/ray/test/run_object_manager_tests.sh
 
-  # cc bazel tests
-  - ./ci/suppress_output bazel test --build_tests_only --show_progress_rate_limit=100 --test_output=errors //:all
+  # cc bazel tests (w/o RLlib)
```

Comment:
Is -rllib needed? It is already ignoring the other py tests.

Reply:
I'm not sure. Not super familiar with bazel yet. Just trying things here and there till they work. :)

Reply:
Yes, I'm also surprised that the main `python/ray/...` bazel test at the bottom of the travis file does NOT run rllib tests, b/c the `rllib/BUILD` file is also inside `python/ray/...`.